Scripts/create-archive.sh

```diff
@@ -98,14 +98,14 @@ function verify_architectures() {
   done
 }
 verify_architectures "iOS/*.framework" armv7 armv7s arm64 arm64e i386 x86_64 || exit $?
-verify_architectures "macOS/*.framework" x86_64 || exit $?
-verify_architectures "tvOS/*.framework" arm64 i386 x86_64 || exit $?
+verify_architectures "macOS/*.framework" arm64 x86_64 || exit $?
+verify_architectures "tvOS/*.framework" arm64 x86_64 || exit $?
 verify_architectures "XCFramework/*.xcframework/ios-!(*-*)/*.framework" armv7 armv7s arm64 arm64e || exit $?
-verify_architectures "XCFramework/*.xcframework/ios-*-maccatalyst/*.framework" x86_64 || exit $?
-verify_architectures "XCFramework/*.xcframework/ios-*-simulator/*.framework" i386 x86_64 || exit $?
-verify_architectures "XCFramework/*.xcframework/macos-*/*.framework" x86_64 || exit $?
+verify_architectures "XCFramework/*.xcframework/ios-*-maccatalyst/*.framework" arm64 x86_64 || exit $?
+verify_architectures "XCFramework/*.xcframework/ios-*-simulator/*.framework" arm64 i386 x86_64 || exit $?
+verify_architectures "XCFramework/*.xcframework/macos-*/*.framework" arm64 x86_64 || exit $?
 verify_architectures "XCFramework/*.xcframework/tvos-!(*-*)/*.framework" arm64 || exit $?
-verify_architectures "XCFramework/*.xcframework/tvos-*-simulator/*.framework" i386 x86_64 || exit $?
+verify_architectures "XCFramework/*.xcframework/tvos-*-simulator/*.framework" arm64 x86_64 || exit $?
```

Comment:
What's with tvOS i386? Is it becsuse of framework changes?

Reply:
it's not in `ARCHS_STANDARD` for tvOS, it was just explicitly copied from iOS Simulators in our build scripts